index.html

```diff
@@ -15,7 +15,7 @@
       <!-- This is the main page -->
       <div id="quiz-header" class="quiz-header">
         <div id="logo" class="logo">
-          <img class="logo-JSQ" src="/client/assets/LogoJSQ.png" alt="logoQuiz">
+          <img class="logo-jsq" src="/client/assets/Logo-jsq.png" alt="logoQuiz">
```

Comment:
just add a period in `src="./client/assets/Logo-jsq.png" ` before the first slash ' / '

Reply:
oh, okay, true, the file was there on the master branch, just not in the bug-fix :) after updating master, it works for me, even without the dot :)